cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -59,6 +60,8 @@ public class Person implements JsonSerializable {
     private Boolean reserved;
     @JsonProperty(EMPLOYMENTS)
     private Set<Employment> employments;
+    @JsonProperty(VERIFIED)
```

Comment:
I am not a fan of this way of defining dto's, but you are following the pattern that was already there,  
please read & ignore.

Reply:
Exactly, did not want to change status quo for now, might be room for refactor later